packages/mw-newrelic/test/mw-newrelic.test.js

```diff
@@ -1,8 +1,37 @@
 /**
  * Created by arolave on 14/05/2017.
  */
-describe('no tests', () => {
-  it('no test', () => {
-    // fail();
+
+import Promise from 'bluebird';
+import set from 'lodash/set';
+import middleware from '../src';
+
+describe('Test NR middleware use cases', () => {
+  it('Should pass request w/o queue to the next midddleware ', () => {
+    const req = {};
+    const val = 'success';
+    const next = () => Promise.resolve(val);
+
+    middleware(req, next)
```

Comment:
you should add a mock for the newrelic lib to show that the calls happen as expected...

Reply:
https://facebook.github.io/jest/docs/en/mock-functions.html

Reply:
New relic without config file acts like a mock (do nothing)
I'll check it anyway